pkg/ddl/index.go

```diff
@@ -770,9 +770,9 @@ func pickBackfillType(ctx context.Context, job *model.Job, unique bool, d *ddlCt
 			if err != nil {
 				return model.ReorgTypeNone, err
 			}
-			if variable.EnableDistTask.Load() {
-				_, err = ingest.LitBackCtxMgr.Register(ctx, unique, job.ID, d.etcdCli, job.ReorgMeta.ResourceGroupName)
-			} else {
+
+			// for dist task, we Register BackCtx in backfillDistScheduler.Init.
+			if !variable.EnableDistTask.Load() {
```

Comment:
We can remove the `Register()` code here. Later I will file a PR to implement "only one task can run at the same time".

Reply:
#48645